dist/.goreleaser.yaml

```diff
@@ -64,6 +64,7 @@ nfpms:
     builds:
       - server
     package_name: "scylla-manager-server"
+    file_name_template: "{{ .ProjectName }}-server-{{ .Version }}_{{ .Os }}.{{ .Arch }}"
```

Comment:
What is the problem these file name template changes are fixing? It would be nicer to consolidate them in the rpm packaging `overrides` section.

Reply:
Since we are aiming to use same code in scylla-pkg. i changed the template to match what we have in `Scylla`

Reply:
Ok, but why does this _template_ have to be defined for very package? Can't you add it to the `override` section once for all packages?

Reply:
There is a different between rpm and deb packages. this is working (since it's identical to what we have in Scylla):
`scylla-manager-agent_2.5-dev-0.20210722.7b12b648.linux_amd64.deb`
`scylla-manager-agent-2.5-dev-0.20210722.7b12b648_linux.x86_64.rpm`

While if i remove the templates i will get:
`scylla-manager-agent_2.5-dev-0.20210722.7b12b648_linux_x86_64.rpm`
`scylla-manager-agent_2.5-dev-0.20210722.7b12b648_linux_amd64.deb`

So removing the templates will make us change the scripts in scylla-pkg and will add more code and complication
@penberg